lib/index.ts

```diff
@@ -282,6 +283,85 @@ export async function getObjectWithLinks<
 	return contractWithLinks as PContract;
 }
 
+// Only org members can add other users to an org
+// Here we check to see if the user is an org member, or the creator of the org, before allowing the link creation
+// TODO: This is a temporary solution until we have a better way to handle this.
+// Once we are able to distinctly permission writes in AutumnDB, we can remove this.
+const validateOrgMembership = async (
+	logContext: LogContext,
+	kernel: Kernel,
+	session: AutumnDBSession,
+	contract: Partial<Contract>,
+): Promise<void> => {
+	if (contract.type === 'link@1.0.0' && session.actor.slug !== 'user-admin') {
```

Comment:
Should we also allow users with the operator role to manage org membership for orgs they don't belong to?

Reply:
I don't think so, do you have a use case that you're thinking of?

Reply:
Not one in particular, just a thought that crossed my mind - is there a case to be made for a role having global org management abilities? From what you have here, I don't think we'll need it, which probably means we should at least start off without it. Just wanted to toss that idea in and see if it made sense.